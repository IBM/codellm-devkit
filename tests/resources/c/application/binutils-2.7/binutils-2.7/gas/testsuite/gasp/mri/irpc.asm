	irpc	dummy,1234
	dc.l	dummy
	endr
