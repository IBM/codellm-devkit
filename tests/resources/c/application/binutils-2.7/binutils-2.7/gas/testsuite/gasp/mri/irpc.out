;	irpc	dummy,1234
;	dc.l	dummy
;	endr
	dc.l	1
	dc.l	2
	dc.l	3
	dc.l	4
;